app/services/template_versioning_service.rb

```diff
@@ -68,8 +68,8 @@ def self.unschedule!(template_version, deprecated_by)
     template_version
   end
 
-  def self.force_publish_now!(requirement_template)
-    unless ENV["ENABLE_TEMPLATE_FORCE_PUBLISH"] == "true"
+  def self.force_publish_now!(requirement_template, for_early_access = false)
+    unless ENV["ENABLE_TEMPLATE_FORCE_PUBLISH"] == "true" || for_early_access
```

Comment:
is `for_early_access` defined?

Reply:
I'm not sure I understand, it's defined as the parameter on line 71?